### DISCUSSION TITLE
admission webhook "validator.longhorn.io" denied the request: spec and status of disks on node <node> are being syncing and please retry later.

### DISCUSSION THREAD

**Discussion by colliwhopper** [longhorn/longhorn#4330] [General]
Hi all, 

no matter what I do, uninstall/reinstall, clear out CRDs/longhorn folders and volumes on the x2 nodes etc etc, the node "kn02" remains uneditable with the below screenshot. If I try to edit the node I get:

"admission webhook "validator.longhorn.io" denied the request: spec and status of disks on node kn02 are being syncing and please retry later."

https://i.postimg.cc/Nf3ZJXZz/Screenshot-2022-07-30-at-20-16-59.png

k8s v1.24.3
longhorn 1.30 and 1.3.1rc1.

Any ideas?

**Comment by derekbit** [comment]
@colliwhopper

It's a known issue and fixed in the upcoming v1.3.1. You can refer to the https://github.com/longhorn/longhorn/issues/4143#issue-1276280630 temporarily.

**Comment by jorikseldeslachts** [comment]
Same issue on k3s v1.25.4+k3s1 with longhorn 1.4.0.  I can't delete or edit the longhorn node anymore.

**Reply by derekbit** [reply]
The issue is already fixed in v1.3.1. Can you provide the support bundle?
